Permanent/ViewControllers/SideMenuViewController.swift

```diff
@@ -82,14 +88,23 @@ class SideMenuViewController: BaseViewController<AuthViewModel> {
     }
     
     func updateLeftSideMenu() {
+        var menuIndexPaths: [IndexPath] = []
+        var menuTitles: [DrawerOption] = []
+        if let selectedArchivePermission = AuthenticationManager.shared.session?.selectedArchive?.permissions(), selectedArchivePermission.contains(.legacyPlanning) {
+            menuIndexPaths = [IndexPath(item: 1, section: 2), IndexPath(item: 2, section: 2), IndexPath(item: 3, section: 2)]
+            menuTitles = [DrawerOption.manageTags, DrawerOption.manageMembers, DrawerOption.legacyPlanning]
```

Comment:
Looks like duplicated code. You could have: 
var menuIndexPaths: [IndexPath] = [IndexPath(item: 1, section: 2), IndexPath(item: 2, section: 2)]
var menuTitles: [DrawerOption] = [DrawerOption.manageTags, DrawerOption.manageMembers]
if viewModel.hasLegacyPermissions() {
menuIndexPaths.append(IndexPath(item: 3, section: 2))
menuTitles.append(DrawerOption.legacyPlanning)
}

Reply:
I will make this change.